internal/MozziGuts.hpp

```diff
@@ -289,9 +294,9 @@ uint32_t MozziRandPrivate::z=521288629;
 unsigned long audioTicks() { return MozziPrivate::audioTicks(); };
```

Comment:
Sorry, but looking at this file (I am still lost in the new organization, too many guts), and this is not related, but why would `unsigned long mozziMicros() { return MozziPrivate::mozziMicros(); };` conflict with `mozziMicros()` but not the other ones?

Reply:
Oh, good catch. I must have commented that line by accident. Fixed (and also clarified the explanation at bit).